File: Expressions/Financial_Calculator.c
// Sasha Rinitra, Financial Calculator C

#include <stdio.h>

char welcome[] = "Welcome to Budget Calculator!";
float income_amount;
float rent_amount;
float utilities_amount;
float groceries_amount;
float transportation_amount;

float percent_rent;
float percent_utilities;
float percent_groceries;
float percent_transportation;
float percent_spending_income;
float percent_savings;

int main(void){

    printf("%s\n", welcome);

    printf("What is your monthly income?\n");
    scanf("%f", &income_amount);

    printf("How much do you pay for rent every month?\n");
    scanf("%f", &rent_amount);

    printf("How much do you spend for utilities every month?\n");
    scanf("%f", &utilities_amount);

    printf("How much do you spend on groceries every month?\n");
    scanf("%f", &groceries_amount);

    printf("How much do you spend on transportation every month?\n");
    scanf("%f", &transportation_amount);

    float savings = income_amount*0.1;
    float income_spending = rent_amount+utilities_amount+groceries_amount+transportation_amount+savings;

    float percent_rent = (rent_amount/income_amount)*100;
    float percent_utilities = (utilities_amount/income_amount)*100;
    float percent_groceries = (groceries_amount/income_amount)*100;
    float percent_transportation = (transportation_amount/income_amount)*100;
    float percent_spending_income = (income_spending/income_amount)*100;
    float percent_savings = (savings/income_amount)*100;

    printf("The amount of money you spend on rent is $%.2f and that is %.2f%% of your income\n", rent_amount, percent_rent);
    printf("The amount of money you spend on utilities is $%.2f and that is %.2f%% of your income\n", utilities_amount, percent_utilities);
    printf("The amount of money you spend on groceries is $%.2f and that is %.2f%% of your income\n", groceries_amount, percent_groceries);
    printf("The amount of money you spend on transportation is $%.2f and that is %.2f%% of your income\n", transportation_amount, percent_transportation);
    printf("The amount of money you spend on a total of $%.2f and that is %.2f%% of your income\n", income_spending, percent_spending_income);
    printf("You save $%.2f which is %.1f%% of your income\n", savings, percent_savings);
    
    return 0;
}

File: Expressions/Financial Calculator.py
# Sasha Rinitra, Financial Calculator Python

# Write a print statement telling the user what the program is (Budget Calculator)
print("Welcome to Budget Calculator!")
# Ask for monthly input (User input)
income_amount = float(input("What is your monthly income\n"))
# Ask for rent/mortgage amount (User input)
rent_amount = float(input("How much do you pay or your rent ever month?\n"))
# Ask for utilities amount (User input)
utilities_amount = float(input("How much do you spend on utilities every month?\n"))
# Ask for groceries amount (User input)
groceries_amount = float(input("How much do you spend on groceries every month?\n"))
# Ask for transportation amount (User input)
transportation_amount = float(input("How much do you spend on your transportation?\n"))
# Calculate savings as 10% income (variable)
savings = income_amount*0.1
# Calculate spending money, income - (rent+utilities+groceries+transportations+savings) (variable)
income_spending = rent_amount+utilities_amount+groceries_amount+transportation_amount+savings
# Calculate percent of rent (rent/income[divide])*100 (variable)
percent_rent = (rent_amount/income_amount)*100
# Calculate percent of utilities (utilities/income[divide])*100 (variable)
percent_utilities = (utilities_amount/income_amount)*100
# Calculate percent of groceries (groceries/income[divide])*100 (variable)
percent_groceries = (groceries_amount/income_amount)*100
# Calculate percent of transportation (transportation/income[divide])*100 (variable)
percent_transportation = (transportation_amount/income_amount)*100
# Calculate percent of spending money (spending/income[divide])*100 (variable)
percent_spending_income = (income_spending/income_amount)*100
# Tell user category spending amount and percent for rent ("You spend", rent, "on rent and that is", percent_rent, "percent of your income")
print("You spend", (f"{rent_amount:.2f}"), "on rent and that is", (f"{percent_rent:.1f}"), "percent of your income")
# Tell user category spending amount and percent for utilities ("You spend $xx.xx on utilities and that is xx% of your income")
print("You spend", (f"{utilities_amount:.2f}"), "on utilities and that is", (f"{percent_utilities:.1f}"), "percent of your income")
# Tell user category spending amount and percent for groceries ("You spend $xx.xx on groceries and that is xx% of your income")
print("You spend", (f"{groceries_amount:.2f}"), "on groceries and that is", (f"{percent_groceries:.1f}"), "percent of your income")
# Tell user category spending amount and percent for transportation ("You spend $xx.xx on transportaion and that is xx% of your income")
print("You spend", (f"{transportation_amount:.2f}"), "on transportation and that is", (f"{percent_transportation:.1f}"), "percent of your income")
# Tell user category spending amount and percent for spending ("You spend $xx.xx on spending money and that is xx% of your income")
print("You spend", (f"{income_spending:.2f}"), "in total and that is", (f"{percent_spending_income:.1f}"), "percent of your income")
# Tell user category spending amount and percent for savings ("You spend $xx.xx on savings and that is 10% of your income")
print("You save", (f"{savings:.2f}"), "and that is 10 percent of your income")